src/app/shell/info/about/about.component.ts

```diff
@@ -26,6 +27,7 @@ export class AboutComponent implements OnInit, OnDestroy {
 
   platformInformation: CompanyInformation;
 
+  readonly emptyList = NoResultsTitle.noInfo;
```

Comment:
move read-only field to the very top (before platformInformation) so we will stay consistent we our code style. it's not a strict rule, just approach for our project